pipelines/run-collectors/run-collectors.yaml

```diff
@@ -94,6 +94,8 @@ spec:
           value: $(params.collectorsRepositoryRevision)
         - name: resultsDir
           value: $(tasks.collect-collectors-resources.results.resultsDir)
+        - name: previousReleasePath
```

Comment:
README entry and version bump for this change missing.

Reply:
thanks. fixed